Core/src/main/java/com/github/intellectualsites/plotsquared/plot/util/SchematicHandler.java

```diff
@@ -152,6 +146,8 @@ public void paste(final Schematic schematic, final Plot plot, final int xOffset,
             }
             try {
                 // Set flags
+                /*
+                TODO: Revisit
```

Comment:
Revisit when? Is this something that will be worked on after the v5 release?

Reply:
Right. We need to determine whether this is something we want to support at all. What is the purpose of storing plot flags in the schematics to begin with? We should discuss this in the developer channel on Discord.

Reply:
I see no reason we should be saving flags inside schematics because it isn't part of the schematic standard and we already have issues with schematics as is.